Cargo.lock

```diff
@@ -119,7 +119,7 @@ name = "bigint"
 version = "3.0.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "byteorder 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "byteorder 1.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
yes, it is needed because 1.0 does not define behavior of `read_f64` to mask out sNaN, while 1.1.0 does.